App.js

```diff
@@ -15,10 +15,9 @@ const sentryApi = secrets[namespace].sentryApi;
 
 if (sentryApi?.dsn) {
   Sentry.init({
-    dsn: sentryApi.dsn,
-    enableNative: true, // NOTE: Native crash reporting is not available with the classic build system (expo build:[ios|android]), but is available via EAS Build.
+    dsn: sentryApi.dsn
     // enableInExpoDevelopment: true, // NOTE: Use this to enable temporarily tracking errors in development
```

Comment:
i cannot find anything about `enableInExpoDevelopment`, seems like this is not existing anymore: https://docs.sentry.io/platforms/react-native/configuration/options/#debug

Reply:
That's right! You're right
https://docs.sentry.io/platforms/react-native/migration/sentry-expo/#review-sentryinit-options

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/954b2437b82a2f81ba0a210663bde9a7c735368b